src/sklearn_raster/features.py

```diff
@@ -138,65 +138,65 @@ def _postprocess_ufunc_output(
         """
 
     @staticmethod
-    def from_image(image: Any, nodata_input: NoDataType = None) -> Image:
-        """Create an Image object from a supported image type."""
-        if isinstance(image, np.ndarray):
-            return NDArrayImage(image, nodata_input=nodata_input)
+    def from_features(features: Any, nodata_input: NoDataType = None) -> FeatureArray:
```

Comment:
I guess if I'm being consistent, I'd argue this method should be called `from_feature_array`.